src/api/supportRequest.ts

```diff
@@ -53,16 +53,28 @@ supportRequestRoutes.get('/getInProgress', async (req, res) => {
 });
 
 supportRequestRoutes.post('/getNext', async (req, res) => {
-  const { adminName } = req.body;
-
+  const { adminName, requestType } = req.body;
   if (!adminName || !adminName.trim()) {
     res.status(400).send("Property 'adminName' is required");
     return;
   }
 
   let nextRequest;
   try {
-    nextRequest = await SupportRequest.getNextSupportRequest();
+    switch (requestType) {
```

Comment:
convert this to a string and use the string values defined within `entities/supportRequest.ts`

Reply:
Changes to the front end are gone  👍